pact.cabal

```diff
@@ -144,55 +144,55 @@ library
   build-depends:       Decimal >= 0.4.2 && < 0.6
                      , aeson >= 0.11.3.0 && < 1.5
                      , algebraic-graphs >= 0.2 && < 0.4
-                     , prettyprinter >= 1.2 && < 1.3
-                     , prettyprinter-ansi-terminal >= 1.1 && < 1.2
-                     , prettyprinter-convert-ansi-wl-pprint
-                     , attoparsec >= 0.13.0.2 && < 0.14
+                     , prettyprinter ^>= 1.2
```

Comment:
I don't have a strong opinion about this one.  If anything I'd say small preference for the old style since that's the direction things will tend to move over time as new major version bumps come out and you want to support them.  But I'll defer to @slpopejoy.